gensim/matutils.py

```diff
@@ -382,6 +383,110 @@ def cossim(vec1, vec2):
     return result
 
 
+def isbow(vec):
+    """
+    Checks if vector passed is in bag of words representation or not.
+    """
+    if scipy.sparse.issparse(vec):
```

Comment:
@piskvorky , I do the sparse check only to make my checking if it's a bow or not easier - so would it be ok for me to keep this? I now do:

`vec = numpy.array(vec.todense())`

before going ahead, so that the rest of the code only deals with either numpy arrays or lists.